modules/serverless-nodejs-functions-context-objects.adoc

```diff
@@ -53,3 +53,64 @@ function handle(context, data)
 ----
 
 The `data` parameter in this example is a JavaScript object that contains the `customerId` and `productId` properties.
+
+[id="serverless-nodejs-functions-context-objects-arbitrary-data_{context}"]
+== Arbitrary data
+
+A function can receive any data, not just CloudEvents. For example, you might want to call a function using POST with an arbitrary object in the body:
+
+[source,json]
+----
+{
+  "id": "12345",
+  "contact": {
+    "title": "Mr.",
+    "firstname": "John",
+    "lastname": "Smith"
+  }
+}
+----
+
+In this case, you can define the function as follows:
+
+[source,javascript]
+----
+function handle(context, customer) {
+  return "Hello " + customer.contact.title + " " + customer.contact.lastname;
+}
+----
+
+Supplying the contact object to the function would then return the following:
```

Comment:
"following" should be an adjective in this case--"following example," or maybe "following example output."